### PR TITLE
{bio}[foss/2018b] VCFtools v0.1.16

### PR DESCRIPTION
**Pull request by smoors**
(created using `eb --new-pr`)

### CODE REVIEW

File: easybuild/easyconfigs/v/VCFtools/VCFtools-0.1.16-foss-2018b-Perl-5.28.0.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+version = '0.1.16'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = "https://vcftools.github.io"
+description = """The aim of VCFtools is to provide 
+ easily accessible methods for working with complex 
+ genetic variation data in the form of VCF files."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'optarch': True}
```
Comment (boegel) [inline easybuild/easyconfigs/v/VCFtools/VCFtools-0.1.16-foss-2018b-Perl-5.28.0.eb]: @smoors No need to enable `optarch`, it's enabled by default, so please remove this line.

**Comment by boegel** [comment]
@smoors I forget about this one when merging #7101, so now this is blocked because of a merge conflict...

I merged #7101 without requiring that enabling `optarch` was removed, because I figured we should do that in one single sweep some time, we shouldn't keep blocking PRs over that.

Of course, this PR should have gone in long before #7101, my apologies for not picking up on it again... :(

**Comment by boegel** [comment]
follow-up on removing `optarch` toolchain option in `VCFtools` easyconfigs via #7106...

**Comment by smoors** [comment]
ok, no problem @boegel

**Comment by verdurin** [comment]
@smoors it's my fault for not checking open PRs as well as `develop` - lesson learned...

**Comment by smoors** [comment]
@verdurin I also often forget to do this, so don't worry :)

**Pull request closed without merging**